### PR TITLE
feat: swap title

### PR DESCRIPTION
**Pull request by kyhyco**
**What changed? Why?**
- allow users to set the title for Swap component

**Notes to reviewers**

**How has it been tested?**
locally

### CODE REVIEW

**Comment by Zizzamia** [comment]
@kyhyco let's take a minute to add a test for this case.

File: src/swap/components/Swap.test.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    useBalance: jest.fn(),
+    useReadContract: jest.fn(),
+  };
+});
+
+jest.mock('../../internal/text', () => {
+  return {
+    TextTitle3: ({ children }: { children: string }) => <div>{children}</div>,
+  };
+});
```
Comment (kyhyco) [inline src/swap/components/Swap.test.tsx:20]: I will fix this in a follow up PR. Getting a bit messy with resolving external dependencies like `clsx`

File: src/swap/components/Swap.tsx [hunk 2]
```diff
@@ -76,6 +78,7 @@ export function Swap({ address, children }: SwapReact) {
     [fromToken, toToken],
   );
 
+  /* istanbul ignore next */
```
Comment (kyhyco) [inline src/swap/components/Swap.tsx:81]: I will add tests for these in a follow up PR

**Review by Zizzamia — APPROVED** [review]
Niiiice

**Pull request merged**